lib/kitchen/directions/web_post_bake_restore.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    module WebPostBakeRestore
+      def self.v1(book_pages:)
+        pantry = book_pages.first.ancestor(:book).pantry(name: :web_placeholders)
+
+        # Replace title & metadata elements, remove placeholders
+        pantry.each do |replacement_key, element|
+          placeholder_element = book_pages.search(
+            "[data-replacement-key='#{replacement_key}']"
+          ).first
+          placeholder_element.append(sibling: element.to_s)
```

Comment:
Sometimes titles contain other elements, like `<sup>`, `<sub>`, `<math>` etc. I'm afraid that converting `element` to string might break an initial form.

Reply:
thanks! i [updated the spec](https://github.com/openstax/cookbook/pull/322/commits/6aa3bdb570c0ac500e9764a33fbb46b99cb619c7) to include nested tags + namespaces. looks like it should work. (i also tested the branch on the full university-physics locally and made sure the diff was 0 between pdf-baked and web-baked when these 2 directions were added in & BakeImages was commented out.)

Reply:
Great to hear that! :tada: